JavaFXUserInterface/src/main/java/Controllers/SystemsController.java

```diff
@@ -96,13 +118,22 @@ public void attachEvents() {
         });
 
         //Attach link to systemMenuButton to go to Systems.fxml
-        systemMenuBtn.setOnMouseClicked(mouseEvent -> uiUtilities.changeScene(mouseEvent, "/Systems"));
+        systemMenuBtn.setOnMouseClicked(mouseEvent -> {
+            closeTimeline();
+            uiUtilities.changeScene(mouseEvent, "/Systems");
+        });
 
         //Attach link to systemTypeMenuBtn to go to Utilities.SystemTypeList.fxml
-        systemTypeMenuBtn.setOnMouseClicked(mouseEvent -> uiUtilities.changeScene(mouseEvent, "/SystemTypeList"));
+        systemTypeMenuBtn.setOnMouseClicked(mouseEvent -> {
+            closeTimeline();
+            uiUtilities.changeScene(mouseEvent, "/SystemTypeList");
+        });
 
         //Attach link to addSystemButton to go to AddSystem.fxml
-        addSystemBtn.setOnMouseClicked(mouseEvent -> uiUtilities.changeScene(mouseEvent, "/AddSystem"));
+        addSystemBtn.setOnMouseClicked(mouseEvent -> {
+            closeTimeline();
+            uiUtilities.changeScene(mouseEvent, "/AddSystem");
+        });
```

Comment:
In the latest PR i added a function in uiUtilities with the signature: changeScene(ArrayList<Timeline> timelines, MouseEvent mouseEvent, String s) 

it's probably a good idea to use that and pass the timeline to the changescene to keep consistency

Reply:
I addressed your concerns.